### PR TITLE
Fix #345

### LINKED ISSUE #345: get-express-paramsを使用
```
./src/web/routes/index.ls:61:           do-login req, req.body.\screen-name, req.body.password, (user) ->
./src/web/routes/resources.ls:60:               | req.query.user? =>
./src/web/routes/resources.ls:61:                       User.find-one { screen-name: req.query.user } (, theme-user) ->
./src/web/routes/resources.ls:74:                       style-name = req.params.0
./src/web/routes/resources.ls:95:               if req.query.user?
./src/web/routes/resources.ls:96:                       User.find-one {screem-name: req.query.user} (, theme-user) ->
./src/web/routes/resources.ls:122:                                              | req.query.user? =>
./src/web/routes/resources.ls:123:                                                      User.find-one {screen-name: req.query.user} (, style-user) ->
./src/web/routes/image.ls:37:           | req.query.blur? =>
./src/web/routes/image.ls:39:                           options = parse-json req.query.blur.replace /([a-zA-Z]+)\s?:\s?([^,}"]+)/g '"$1":$2'
./src/web/routes/image.ls:149:          id-or-sn = req.params.idorsn
./src/web/routes/image.ls:154:          id-or-sn = req.params.idorsn
./src/web/routes/image.ls:159:          id-or-sn = req.params.idorsn
./src/web/routes/image.ls:164:          id = req.params.id
./src/web/routes/image.ls:169:          id = req.params.id
./src/web/routes/image.ls:174:          id = req.params.id
./src/web/controllers/dev-usertheme.ls:7:               (callback) -> webtheme.find req.query.q, (themes) -> callback null themes
./src/web/controllers/user-talk.ls:44:                                          no-header: req.query.noheader == \true
./src/web/controllers/dev-myapp.ls:7:           (callback: any) -> Application.find req.query.q, (app) -> callback null app
./src/api/rest/status/update.ls:17:     text = if req.body.text? then req.body.text else ''
./src/api/rest/status/update.ls:18:     in-reply-to-status-id = req.body\in-reply-to-status-id ? null
./src/api/rest/users/follow.ls:10:      | !(target-user-id = req.body\user-id)? => res.api-error 400 'user-id parameter is required :('
./src/api/rest/users/unfollow.ls:10:    | !(target-user-id = req.body\user-id)? => res.api-error 400 'user-id parameter is required :('
./src/api/rest/users/show.ls:8:         user-id = req.body.user_id
./src/api/rest/circle/update.ls:13:     | _ => Circle.find req.body.circle_id, (circle) ->
./src/api/rest/circle/update.ls:17:                     circle.name = req.body.name if req.body.name?
./src/api/rest/circle/update.ls:18:                     circle.description = req.body.description if req.body.description?
./src/api/rest/talk/fix.ls:9:   | !(text = req.body.text)? => res.api-error 400 'text parameter is required :('
./src/api/rest/talk/fix.ls:10:  | !(message-id = req.body.message-id)? => res.api-error 400 'messageId parameter is required :('
./src/api/rest/talk/delete.ls:8:        | !(message-id = req.body.message-id)? => res.api-error 400 'messageId parameter is required :('
./src/api/rest/talk/read.ls:6:  msg-id = req.body.message-id
./src/api/rest/account/update-webtheme.ls:6:    theme-id = req.body.theme-id
./src/api/rest/application/delete.ls:8: | !(id = req.body.id)? => res.api-error 400 'id parameter is required :('
```

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/api/rest/status/update.ls [hunk 1]
```diff
@@ -2,20 +2,22 @@ require! {
...
 	'../../../models/user-following': UserFollowing
-	'../../../utils/publish-redis-streaming'
-	'../../auth': authorize
 }
 
 module.exports = (req, res) -> authorize req, res, (user, app) -> Status.find-one { user-id: user.id }, (, status) ->
-	text = if req.body.text? then req.body.text else ''
-	in-reply-to-status-id = req.body\in-reply-to-status-id ? null
+	[text, in-reply-to-status-id] = get-express-params req, <[ text, in-reply-to-status-id ]>
+	text = if !empty text then text else ''
```
Comment (AyaMorisawa) [inline src/api/rest/status/update.ls]: これ何の意味があります？

Comment (ghost) [inline src/api/rest/status/update.ls]: しまった

**Comment by ghost** [comment]
I fixed.

**Comment by syuilo** [comment]
Message:
    Parse error on line 66: Unexpected 'CASE'
at /home/travis/build/syuilo/Misskey/src/web/routes/resources.js

File: src/web/controllers/user-talk.ls [hunk 2]
```diff
@@ -24,12 +24,11 @@ module.exports = (req, res) ->
...
 						->
-			
 			serialize-talk-messages messages, me, otherparty .then (messages) ->
 				generate-user-talk-message-stream-html messages, me .then (message-htmls) ->
 					res.display req, res, \user-talk {
 						otherparty
 						messages: message-htmls
 						following-me
-						no-header: req.query.noheader == \true
+						no-header: noheader == \true
```
Comment (syuilo) [inline src/web/controllers/user-talk.ls]: ？

**Comment by ghost** [comment]
I fixed it. I'm sorry.

**Comment by AyaMorisawa** [comment]
Please merge syuilo/develop

**Pull request closed without merging**